### ISSUE TITLE
After updating to the latest version, the plugin stopped working.

### ISSUE THREAD

**Issue by mingleeShade** [ojroques/vim-oscyank#39]
I updated the plugin to the latest commit on the master branch and noticed that when copying, a string similar to Base64 encryption appeared on the screen. However, after refreshing the screen, the string disappeared. Additionally, I am unable to copy the string to the windows clipboard using osc52.
I am working on a Windows machine and connecting to a remote Linux machine via SSH. I am using tmux to manage multiple screens.
The plugin was working fine on commit e6298736a7835bcb365dd45a8e8bfe86d935c1f8, and my tmux settings are correct. What could be the problem?
[
![微信图片_20230316103157](https://user-images.githubusercontent.com/15226636/225495036-aa396b6e-0e1d-4292-9d5a-090eb38927ac.png)
](url)
![20230316-103414](https://user-images.githubusercontent.com/15226636/225495400-7ba562a4-5708-48ec-b4e0-0d560100e62e.gif)

**Comment by ojroques** [comment]
Hello!
I refactored the code recently, sorry for the disagreement. Can you share your tmux version, the terminal emulator you're using and your vim config?

In the meantime, you can checkout the `v1.0.0` tag which is just before the refactoring.

**Comment by nbn22385** [comment]
Same issue began happening for me too after updating, connecting to an Ubuntu 22.04 VM via SSH:
- Windows 10
- Windows Terminal 1.16
- tmux 3.2a
- vim 9
- [vim config + dotfiles](https://github.com/nbn22385/dotfiles)

Reverted to tag `v1.0.0` and working again, thanks.

**Comment by ojroques** [comment]
Try adding `set -s set-clipboard on` to your tmux config @nbn22385.

**Comment by mingleeShade** [comment]
I use secureCRT connecting to an Debian9 via SSH, also use tmux as terminal multiplexer:
- windows10
- tmux next-3.1
- neovim 0.9.0
- vim config: [mingleeShade/mynvim](https://github.com/mingleeShade/mynvim)

I already add `set -g set-clipboard on` in my tmux config.

**Comment by ojroques** [comment]
Are you sure secureCRT supports OSC52? Also can you try with `set -s set-clipboard on`?

**Comment by mingleeShade** [comment]
Yes, I am sure that my secureCRT supports OSC52. I have also tried with `set -s set-clipboard on` and the issue persists. Additionally, I have
used an older version of 'vim-oscyank' and it worked without any problems.

**Comment by andrescuco** [comment]
Same issue here, using the older version now as a bandaid fix:

```.vimrc
Plug 'ojroques/vim-oscyank', { 'commit': 'e6298736a7835bcb365dd45a8e8bfe86d935c1f8' }
```

**Comment by AnastasiusVivaldus** [comment]
I too had to revert to v1.0.0 to get it working again. TMUX 3.1b, Ubuntu 20.04, VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Dec 19 2021 20:21:31)

**Comment by ojroques** [comment]
For older versions of tmux, it may work with the option `let g:oscyank_osc52 = "\x1bPtmux;\x1b\x1b]52;c;%s\x07"` (see https://github.com/tmux/tmux/issues/2254) in Vim config but I haven't tried it myself.
